File: trail1/배수의 개수/number-of-multipliers.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    int m=0,n=0;
    for(int i=0;i<10;i++){
        cin>>a;
        if(a%3==0) m++;
        if(a%5==0) n++;
    }

    cout<<m<<" "<<n;
    
    return 0;
}

File: trail1/배수의 개수/number-of-multipliers.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner scin = new Scanner(System.in);

        int c=0,t=0;

        for(int i=0;i<10;i++){
            int a=scin.nextInt();

            if(a%3==0) c++;
            if(a%5==0) t++;
        }

        System.out.print(c+" "+t);
    }
}